src/components/PlioQuestion.vue

```diff
@@ -88,6 +93,11 @@
 
 <script>
 import LoadingSpinner from "./LoadingSpinner.vue";
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faWindowClose } from '@fortawesome/free-solid-svg-icons/faWindowClose';
+import { faCheckCircle } from '@fortawesome/free-solid-svg-icons/faCheckCircle';
+import { faTimesCircle } from '@fortawesome/free-solid-svg-icons/faTimesCircle';
+library.add(faWindowClose, faCheckCircle, faTimesCircle);
```

Comment:
This is where we import the library, import the respective icons and add them to the library. They're automatically converted to components because we have registered the component in `main.js` file